File: VS_Code/C/Total_Salary.c
#include<stdio.h>
void main()
{ 
    int Basic_Salary, HRA, DRA, Total_Salary;

    printf("Enter Basic Salary:");
    scanf("%d",&Basic_Salary);

    HRA=Basic_Salary*20/100;
    DRA=Basic_Salary*150/100;

    Total_Salary=Basic_Salary+HRA+DRA;

    printf("Total Salary is:" "%d",Total_Salary);
}

File: VS_Code/Python/Total_Salary.py
Basic_Pay=int(input("Enter Basic Pay:"));

HRA=Basic_Pay*(10/100);
TA=Basic_Pay*(5/100);

Gross_Salary=Basic_Pay+HRA+TA;

Professional_Tax=Gross_Salary*(2/100);

Net_Salary=Gross_Salary-Professional_Tax;
print("Net Salary is:",Net_Salary);